### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by NathanBaulch**
Just thought I'd contribute some typo fixes I stumbled upon. Nothing controversial (hopefully), just 72 simple fixes.

Use the following command to get a quick and dirty summary of the specific corrections made:
```shell
git diff HEAD^! --word-diff-regex='\w+' -U0 \
  | grep -E '\[\-.*\-\]\{\+.*\+\}' \
  | sed -r 's/.*\[\-(.*)\-\]\{\+(.*)\+\}.*/\1 \2/' \
  | sort | uniq -c | sort -n
```

FWIW, the top typos are:
* invokable (16)
* subsciption (15)
* decison (7)
* reques (3)
* dissallow (3)
* documenation (3)
* locahost (3)
* prerequistes (2)
* assinged (2)
* extenions (2)

### CODE REVIEW

File: internal/globalgates/globalgates.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ var NoopTracerProvider = featuregate.GlobalRegistry().MustRegister("service.noop
 	featuregate.WithRegisterToVersion("v0.109.0"),
 	featuregate.WithRegisterDescription("Sets a Noop OpenTelemetry TracerProvider to reduce memory allocations. This featuregate is incompatible with the zPages extension."))
 
-// UseLocalHostAsDefaultHostfeatureGate is the feature gate that controls whether
+// UseLocalHostAsDefaultHostFeatureGate is the feature gate that controls whether
 // server-like receivers and extensions such as the OTLP receiver use localhost as the default host for their endpoints.
```
Comment (songy23) [inline internal/globalgates/globalgates.go:19]:
```suggestion
```

This can be simply removed

File: semconv/v1.27.0/generated_attribute_group.go [hunk 2]
```diff
@@ -810,7 +810,7 @@ const (
 	// Stability: experimental
 	// Examples: 'arn:aws:lambda:REGION:ACCOUNT_ID:function:my-function', '//run.googl
 	// eapis.com/projects/PROJECT_ID/locations/LOCATION_ID/services/SERVICE_ID', '/sub
-	// scriptions/<SUBSCIPTION_GUID>/resourceGroups/<RG>/providers/Microsoft.Web/sites
+	// scriptions/<SUBSCRIPTION_GUID>/resourceGroups/<RG>/providers/Microsoft.Web/sites
```
Comment (codeboten) [inline semconv/v1.27.0/generated_attribute_group.go:813]: If you haven't already, I'd suggest reviewing the semantic conventions repo (https://github.com/open-telemetry/semantic-conventions) as the source of these generated files lives there

**Review by codeboten — COMMENTED** [review]
Thanks for the fixes, just a couple of suggestions

File: semconv/v1.27.0/generated_attribute_group.go (hunk 2 above)
Comment (NathanBaulch) [reply semconv/v1.27.0/generated_attribute_group.go:813]: I have indeed, about a month ago. https://github.com/open-telemetry/semantic-conventions/pull/1379

**Review by codeboten — APPROVED** [review]

**Pull request merged**